Formula/httpd.rb

```diff
@@ -18,14 +18,188 @@ class Httpd < Formula
   depends_on "pcre"
 
   def install
-    # use Slackware-FHS layout as it's closest to what we want.
-    # these values cannot be passed directly to configure, unfortunately.
-    inreplace "config.layout" do |s|
-      s.gsub! "${datadir}/cgi-bin", "#{pkgshare}/cgi-bin"
-      s.gsub! "${datadir}/error",   "#{pkgshare}/error"
-      s.gsub! "${datadir}/htdocs",  "#{pkgshare}/htdocs"
-      s.gsub! "${datadir}/icons",   "#{pkgshare}/icons"
-    end
+    # point config files to opt_prefix instead of the version-specific prefix
+    inreplace "Makefile.in",
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
```

Comment:
What does this change i.e. which output file(s)?

Reply:
It fixes the ServerRoot line in the default `etc/httpd/httpd.conf` configuration.

Reply:
Ok. It looks like it doesn't fix everything in `etc` and it'd be good to do that if possible.